src/librbd/WatchNotifyTypes.h

```diff
@@ -200,11 +200,12 @@ struct ResizePayload : public AsyncRequestPayloadBase {
   static const NotifyOp NOTIFY_OP = NOTIFY_OP_RESIZE;
   static const bool CHECK_FOR_REFRESH = true;
 
-  ResizePayload() : size(0) {}
-  ResizePayload(uint64_t size_, const AsyncRequestId &id)
-    : AsyncRequestPayloadBase(id), size(size_) {}
+  ResizePayload() : size(0), can_shrink(true) {}
+  ResizePayload(uint64_t size_, bool can_shrink_, const AsyncRequestId &id)
+    : AsyncRequestPayloadBase(id), size(size_), can_shrink(can_shrink_) {}
 
   uint64_t size;
+  bool can_shrink;
```

Comment:
Updated the inherited methods. Should I bump up the version in this file or some other location?